src/web/app/components/session-edit-form/session-edit-form.component.html

```diff
@@ -182,11 +182,18 @@ <h5 class="text-md-right">Or: </h5>
             </div>
             <div class="row">
               <a class="pl-3 pt-2"
+                *ngIf="formMode !== SessionEditFormMode.ADD"
                 tmRouterLink="/web/instructor/sessions/individual-extension"
                 [queryParams]="{ courseid: model.courseId, fsname: model.feedbackSessionName }">
                 Individual Deadline Extensions
                 <i class="fas fa-edit"></i>
               </a>
+              <a class="pl-3 pt-2 disabled-link"
```

Comment:
Add the `ngb-tooltip-class` here. We add a dotted underline style to any text with tooltips.